controller/getUser.js

```diff
@@ -0,0 +1,23 @@
+const { default: async } = require("async");
+const data = require('../model/user')
+const jwt = require('jsonwebtoken');
+const giveData = async(req,res) =>{
+    try{
+        console.log(req.query)
+        let decoded = jwt.verify(req.query.id, 'abcd');
```

Comment:
We are not going to fetch token from query string, we will fetch token from header and then will verify it. Also create a middleware to verify JWT.